config/deploy/manfiests.yaml

```diff
@@ -372,6 +372,9 @@ spec:
         - --leader-elect
         command:
         - /manager
+        env:
+        - name: RELATED_IMAGE_LIMITADOR
```

Comment:
Maybe rename it to something like `LIMITADOR_SERVICE_IMAGE` or `LIMITADOR_INSTANCE_IMAGE` or just `LIMITADOR_IMAGE` ... regarding the last one, we could make the distinction with `LIMITADOR_OPERATOR_IMAGE` if needed

Reply:
RELATED_IMAGE prefix is commonly in [OSBS](https://osbs.readthedocs.io/en/latest/users.html?highlight=RELATED_IMAGE#creating-the-relatedimages-section) for productization purposes. 

I am preparing it for the future

Reply:
Actually, the `relatedImages` section in the bundle is automatically created because of that prefix is being used.